daffodil-runtime2/src/main/resources/org/apache/daffodil/runtime2/c/libcli/version.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef VERSION_H
+#define VERSION_H
+
+/**
+ * This file will be overwritten with the correct program version informaion
+ * when code is generated. This exists here only to support compiling the
+ * source during Daffodil builds without generated code to ensure there are no
+ * compilation errors.
+ */
```

Comment:
Let's use // to be consistent with comments in all other C files, and let's tweak what you said a tiny bit :).

```c
// This file will be overwritten with the correct program version information
// when code is generated. This file exists here to support compiling the other
// C files without generated code during Daffodil builds to ensure there are no
// compilation errors in these C files.
```

Reply:
Thanks! Fixed.